docs/platforms/react-native/tracing/instrumentation/automatic-instrumentation.mdx

```diff
@@ -112,12 +112,19 @@ Sentry.init({
 
 ### tracePropagationTargets
 
-The default value of `tracePropagationTargets` is `['localhost', /^\//]`. The React Native SDK will attach the `sentry-trace` header to all outgoing XHR/fetch requests whose destination contains a string in the list or matches a regex in the list. If your frontend is making requests to a different domain, you'll need to add the domain there to propagate the `sentry-trace` header to the backend services, which is required to link transactions together as part of a single trace. **The `tracePropagationTargets` option matches against the entire request URL, not just the domain. Using stricter regex to match certain parts of the URL ensures that requests don't unnecessarily have the `sentry-trace` header attached.**
+The default value of `tracePropagationTargets` is `[/.*/]` for mobile and `['localhost', /^\//]` for web. The React Native SDK will attach the `sentry-trace` header to all outgoing XHR/fetch requests on mobile.
+On web, trace data is only attached to outgoing requests that contain `localhost` in their URL or requests whose URL starts with a `'/'` (for example `GET /api/v1/users`).
+
+<Expandable title="Using React Native on Web? Here is how `tracePropagationTargets` changes.">
```

Comment:
l: could we use a browser section here `<PlatformCategorySection supported={["browser"]}>` like in other docs?